app/views/Performance/performance.html

```diff
@@ -23,6 +23,12 @@ <h2 class="page-title">{{performance.title}}</h2>
                         <h3>{{performance.title}}</h3>
                         <p>{{date.time}}</p>
                     </div>
+                    <span>{{performanceEvents[$index].venue}}</span>
+                    <div class="block-description">
+                        <a ng-click="selectPerfomance($index)">
+                        <span >Придбати квиток</span>
```

Comment:
As I understand, titles should be moved to file. As there are few languages. ex. `{{perfomance.performance.title}}`